pkg/network/admit/admit.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-package admitters
+package admit
```

Comment:
I would recommend calling this `admitter`. Currently it is a verb.

Reply:
done.